### DISCUSSION TITLE
Performance testing

### DISCUSSION THREAD

**Discussion by domw95** [adamb314/ServoProject#21] [General]
This is a great project, always something I wanted to have a go at but haven't found the time. Absolutely love the theory section especially.
I have previously done some testing with hobby servos (https://domwil.co.uk/posts/servo-performance/) to measure their dynamic performance (bandwidth, response time etc), is this something you have considered doing as a before/after to see how much better you have made it? It would be fascinating to see.

**Comment by adamb314** [comment]
I haven't planed on comparing the bandwidth and response time, but I did a comparison of the control error in a previous video:
https://youtu.be/ECLrLupFW10?si=KYO0Stny1fsPWxmD
if you haven't seen it :)

Might also add some more comparison and results in a separate doc file in the future.
